<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chifoumivue</class>
 <widget class="QMainWindow" name="chifoumivue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>563</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>568</width>
    <height>563</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>50000</width>
    <height>50000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0">
          <item row="5" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="ScoreMachine">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="TexteTemps">
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Temps Restant :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="Objectif">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Gagnant Ã </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="Machine">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Machine</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2" alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="SelectMachine">
            <property name="minimumSize">
             <size>
              <width>111</width>
              <height>110</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>111</width>
              <height>110</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ressourcesChifoumi.qrc">
              <normaloff>:/chifoumi/images/rien.gif</normaloff>:/chifoumi/images/rien.gif</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>111</width>
              <height>110</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="Joueur">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Vous</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="SelectJoueur">
            <property name="minimumSize">
             <size>
              <width>111</width>
              <height>110</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>111</width>
              <height>110</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ressourcesChifoumi.qrc">
              <normaloff>:/chifoumi/images/rien.gif</normaloff>:/chifoumi/images/rien.gif</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>111</width>
              <height>110</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="ScoreObjectif">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>5</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="ScoreJoueur">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1" alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="Fight">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>65</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>65</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ressourcesChifoumi.qrc">
              <normaloff>:/chifoumi/images/versus.gif</normaloff>:/chifoumi/images/versus.gif</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>65</width>
              <height>65</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="Temps">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Choisissez une figure :</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="Ciseaux">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>111</width>
               <height>110</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>111</width>
               <height>110</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="ressourcesChifoumi.qrc">
               <normaloff>:/chifoumi/images/ciseau.gif</normaloff>:/chifoumi/images/ciseau.gif</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>111</width>
               <height>110</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Feuille">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>111</width>
               <height>110</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>111</width>
               <height>110</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="ressourcesChifoumi.qrc">
               <normaloff>:/chifoumi/images/papier.gif</normaloff>:/chifoumi/images/papier.gif</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>111</width>
               <height>110</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Pierre">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>111</width>
               <height>110</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>111</width>
               <height>110</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="ressourcesChifoumi.qrc">
               <normaloff>:/chifoumi/images/pierre.gif</normaloff>:/chifoumi/images/pierre.gif</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>111</width>
               <height>110</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0,0">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Pause">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BoutonPartie">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Nouvelle Partie</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="Parametres"/>
    <addaction name="Quitter"/>
   </widget>
   <widget class="QMenu" name="menuaide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="separator"/>
    <addaction name="AProposDe"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuaide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="AProposDe">
   <property name="text">
    <string>A propos de</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="Parametres">
   <property name="text">
    <string>ParamÃ¨tres</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="Quitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressourcesChifoumi.qrc"/>
 </resources>
 <connections/>
</ui>
